backend/s3_backend.go

```diff
@@ -3,6 +3,7 @@ package backend
 import (
 	"fmt"
 	"io"
+	"io/ioutil"
```

Comment:
go imports?

Reply:
It's used here: https://github.com/stripe/sequins/pull/148/files#diff-46f6ba2d612aec109d60649c7f7a9268R231, I don't think it would have compiled otherwise